Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'gds-api-adapters', '40.1.0'
 gem 'airbrake', '4.0.0'
 gem 'logstasher', '0.6.2'
 
-gem 'govuk_navigation_helpers', '~> 3.1'
+gem 'govuk_navigation_helpers', '~> 3.1.1'
```

Comment:
Argh, I misremembered how `~>` works. I thought it always worked at a patch level, so `~> 3.1` and `~> 3.1.1` would both only allow bundler to install `3.1.x`, but the former actually allows `3.x`.